airbyte-webapp/src/pages/routes.tsx

```diff
@@ -43,6 +44,7 @@ const MainViewRoutes: React.FC<{ workspace: WorkspaceRead }> = ({ workspace }) =
           <Route path={`${RoutePaths.Source}/*`} element={<SourcesPage />} />
           <Route path={`${RoutePaths.Connections}/*`} element={<ConnectionPage />} />
           <Route path={`${RoutePaths.Settings}/*`} element={<SettingsPage />} />
+          <Route path={`${RoutePaths.Builder}/*`} element={<BuilderPage />} />
```

Comment:
Since I only added this path to OSS and not to Cloud, this route will not work on Cloud. Because of this, a new Feature didn't seem necessary, but lmk if you think we still need that.

Reply:
In it's current state, I don't think it is useful for any users, since it's really just a yaml text editor with a download button. So I think butting it behind a disabled feature makes sense; I'll make that change

Reply:
Done: https://github.com/airbytehq/airbyte/pull/17482/commits/35cc0be85a856927a446e63b7ac77ba8d56962c0

But @timroes does this mean that in order for me to see the builder UI while I am developing it, I'll just need to add `FeatureItem.AllowBuilder` to `src/hooks/services/Feature/constants.ts` whenever I am developing, and just try to make sure I don't commit those changes to git? Or is there some better way to enable this feature during development?